controllers/tab/settings/appearance/form/PKPAppearanceForm.inc.php

```diff
@@ -39,6 +39,11 @@ function PKPAppearanceForm($wizardMode = false, $additionalSettings = array()) {
 
 		AppLocale::requireComponents(LOCALE_COMPONENT_APP_COMMON);
 
+		$themes = PluginRegistry::getPlugins('themes');
```

Comment:
I suppose my thinking was that `getPlugins()` would be the recommended usage, but we could make a special call to `loadCategory` when we explicitly wanted to reload. But we can chat more about this later.